Packs/AzureStorage/Integrations/AzureStorage/AzureStorage.py

```diff
@@ -0,0 +1,778 @@
+                properties['lastAccessTimeTrackingPolicy']['enable'] = \
+                    args.get('last_access_time_tracking_policy_enabled') == 'true'
+
+            if 'last_access_time_tracking_policy_blob_types' in args:
+                properties['lastAccessTimeTrackingPolicy']['blobType'] = \
+                    args['last_access_time_tracking_policy_blob_types'].split(',')
+
+            if 'last_access_time_tracking_policy_days' in args:
+                properties['lastAccessTimeTrackingPolicy']['trackingGranularityInDays'] = \
+                    args.get('last_access_time_tracking_policy_days')
+
+        if 'restore_policy_enabled' in args or 'restore_policy_min_restore_time' in args \
+                or 'restore_policy_days' in args:
+            properties['restorePolicy'] = {}
+
+            if 'restore_policy_enabled' in args:
+                properties['restorePolicy']['enabled'] = args.get('restore_policy_enabled') == 'true'
+
+            if 'restore_policy_min_restore_time' in args:
+                properties['restorePolicy']['minRestoreTime'] = args.get('restore_policy_min_restore_time')
+
+            if 'restore_policy_days' in args:
+                properties['restorePolicy']['days'] = args.get('restore_policy_days')
+
+        return self.ms_client.http_request(
+            method='PUT',
+            url_suffix=f'/resourceGroups/{self.resource_group_name}/providers/Microsoft.Storage/storageAccounts/'
+                       f'{args["account_name"]}/blobServices/default',
+            params={
+                'api-version': API_VERSION,
+            },
+            json_data={'properties': properties}
+        )
+
+    def storage_blob_containers_create_request(self, args):
+        """
+                Send the user arguments for the create blob container in the request body to the API.
+            Args:
+                args: The user arguments.
+
+            Returns:
+                The json response from the API call.
+        """
+        properties = {}
+
+        if 'default_encryption_scope' in args:
+            properties['defaultEncryptionScope'] = args.get('default_encryption_scope')
+
+        if 'deny_encryption_scope_override' in args:
+            properties['denyEncryptionScopeOverride'] = argToBoolean(args.get('deny_encryption_scope_override'))
+
+        if 'public_access' in args:
+            properties['publicAccess'] = args.get('public_access')
+
+        return self.ms_client.http_request(
+            method='PUT',
+            url_suffix=f'/resourceGroups/{self.resource_group_name}/providers/Microsoft.Storage/storageAccounts/'
+                       f'{args["account_name"]}/blobServices/default/containers/{args["container_name"]}',
+            params={
+                'api-version': API_VERSION,
+            },
+            json_data={'properties': properties}
+        )
+
+    def storage_blob_containers_update_request(self, args):
```

Comment:
we can unite this method with `storage_blob_containers_create_request` and simply pass the `method` as a n argument to the function depending from which command it came.

Reply:
done.